deploy/roles/intellij/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Download intellij
+  get_url: 
+    url: "{{ intellij_url }}"
+    dest: "/tmp/intellij-{{ intellij_version }}"
+  tags: 
+  - intellij
```

Comment:
So are we not using the RPM that you built to install IntelliJ?  I think this Ansible role should support that option as well.  There is a variable called "custom_repo" that is True in our 3 environments, and False by default when using Vagrant.  You should have one install command that uses the RPM, and is only run when "custom_repo" is True, and these other commands should only be run when "custom_repo" is False.